tests/scipy_signal_test.py

```diff
@@ -100,11 +100,11 @@ def testConvolutions(self, xshape, yshape, dtype, mode, op):
     ],
     dtype=default_dtypes,
   )
-  def testFFTConvolution(self, shape, dtype):
+  def testFFTConvolution(self, xshape, yshape, dtype):
     jsp_op = jsp_signal.fftconvolve
     osp_op = osp_signal.fftconvolve
     rng = jtu.rand_default(self.rng())
-    args_maker = lambda: [rng(shape[0], dtype), rng(shape[1], dtype)]
+    args_maker = lambda: [rng(xshape[0], dtype), rng(yshape[1], dtype)]
```

Comment:
I think this should be `xshape` and `yshape` rather than `xshape[0]` and `yshape[1]`.

`shape[0]` and `shape[1]` were appropriate before when you had `shape = (xshape, yshape)`.

Reply:
ho! yes done

Reply:
tests ok